app/assets/javascripts/discourse/controllers/insert-hyperlink.js.es6

```diff
@@ -1,15 +1,130 @@
 import ModalFunctionality from "discourse/mixins/modal-functionality";
+import { searchForTerm } from "discourse/lib/search";
+import { observes } from "ember-addons/ember-computed-decorators";
 
-export default Ember.Controller.extend(ModalFunctionality, {
-  linkUrl: "",
-  linkText: "",
+let searchTopics;
```

Comment:
You should make sure this is cleaned after tests, otherwise we carry this forever doing tests.